src/components/formio/textfield/textfield.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import {ITextFieldComponent, TextField} from '@components';
+import {IRenderComponentProps, RenderComponent} from '@lib/renderer';
+import {expect} from '@storybook/jest';
+import type {ComponentStory, Meta} from '@storybook/react';
+import {userEvent, within} from '@storybook/testing-library';
+import {Formik} from 'formik';
+import React from 'react';
+
+const meta: Meta<typeof TextField> = {
+  title: 'Components / Formio / Textfield',
+  component: TextField,
+  decorators: [],
+  parameters: {},
+};
+export default meta;
+
+export const textfield: ComponentStory<
+  React.FC<IRenderComponentProps<ITextFieldComponent>>
```

Comment:
not sure if it's possible, but can't you use some `typeof ...` stuff here to avoid re-declaring those types? That would make it a bit more self-correcting and more DRY.